morpheus/stages/preprocess/deserialize_stage.py

```diff
@@ -63,8 +62,7 @@ def __init__(self,
                  c: Config,
                  *,
                  ensure_sliceable_index: bool = True,
-                 message_type: typing.Union[typing.Literal[MultiMessage],
-                                            typing.Literal[ControlMessage]] = MultiMessage,
+                 message_type: typing.Union[MultiMessage, ControlMessage] = MultiMessage,
```

Comment:
This means an instance of either type. I think you want to use `type[MultiMessage] | type[ControlMessage]`